### ISSUE TITLE
Fix type definitions for NPM package

### ISSUE THREAD

**Issue by XR4Z0R** [gildas-lormeau/zip.js#214]
Hi there,

I would like to use zip.js in a TypeScript project. When importing the module I noticed the compiler complains about the type definitions or the module in general. This issue seems to relate to #191 (maybe also #212?)
Adapting the `tsconfig.json` file in my project as shown in the referenced issue did work. But I had a look at the NPM module of zip.js and noticed that changing
```typescript
declare module "zip.js" {
```
to
```typescript
declare module "@zip.js/zip.js" {
```
in the `index.d.ts` file would resolve the compiler issues without any adaptions to the TypeScript config and make it possible to import the module as `"@zip.js/zip.js"` without any path aliases.

I was wondering whether this would be a valid solution to the TypeScript issues or whether I missed something that would open other importing or bundling issues at another point.

Kind regards

**Comment by gildas-lormeau** [comment]
Thank you for having reported the issue. Actually, I did not test the type definitions yet (I code in JS). I'm going to apply the change you propose. You can submit a PR if you want.

**Comment by gildas-lormeau** [comment]
I published the new version 2.2.2 on NPM with your fix.
